src/omv/sensors/paj6100.c

```diff
@@ -0,0 +1,788 @@
+/*
+ * File: paj6100.c
+ * Created Date: Tuesday, May 25th 2021, 10:45:35 am
+ * Author: Lake Fu
+ * -----
+ * Last Modified: Tuesday June 15th 2021 2:54:43 pm
+ * Modified By: Lake Fu at <lake_fu@pixart.com>
+ * -----
+ * MIT License
+ *
+ * Copyright (c) 2021 Pixart Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+ * of the Software, and to permit persons to whom the Software is furnished to do
+ * so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ * -----
+ * HISTORY:
+ * Date      	By	Comments
+ * ----------	---	----------------------------------------------------------
+ */
+#include "omv_boardconfig.h"
+#include "paj6100.h"
+#if (OMV_ENABLE_PAJ6100 == 1)
+#include <stdio.h>
```

Comment:
```suggestion
#include <stdio.h>
#include <stdbool.h>
```

Reply:
I need stdbool.h in the header, so it is not reasonable to remove it to body.
(The function declare use "bool")